package.json

```diff
@@ -51,7 +51,7 @@
     "source-map-support": "^0.4.6",
     "sqlite3": "3.1.8",
     "universal-analytics": "^0.4.8",
-    "uuid": "^3.0.0",
+    "uuid": "^3.1.0",
```

Comment:
This happened because I ran `yarn add uuid` but I think it's not worth reverting :)